deploy-web/src/utils/sdl/sdlGenerator.ts

```diff
@@ -32,10 +35,23 @@ export const generateSdl = (formData: SdlBuilderFormValues) => {
         const to = e.to.map(to => ({ ["service"]: to.value }));
         _expose["to"] = [
           {
-            global: !!e.global
+            global: !!e.global,
+            ...(e.ipName && { ip: e.ipName })
           }
         ].concat(to as any);
 
+        // HTTP Options
+        if (e.hasCustomHttpOptions) {
+          _expose["http_options"] = {
+            max_body_size: e.httpOptions?.maxBodySize || defaultHttpOptions.maxBodySize,
+            read_timeout: e.httpOptions?.readTimeout || defaultHttpOptions.readTimeout,
+            send_timeout: e.httpOptions?.sendTimeout || defaultHttpOptions.sendTimeout,
+            next_cases: e.httpOptions?.nextCases || defaultHttpOptions.nextCases,
+            next_tries: e.httpOptions?.nextTries || defaultHttpOptions.nextTries,
+            next_timeout: e.httpOptions?.nextTimeout || defaultHttpOptions.nextTimeout
```

Comment:
Since some of those `httpOptions` are of type `number`, should we be using `??` instead of `||` so that a value of `0` is kept?
```suggestion
            max_body_size: e.httpOptions?.maxBodySize ?? defaultHttpOptions.maxBodySize,
            read_timeout: e.httpOptions?.readTimeout ?? defaultHttpOptions.readTimeout,
            send_timeout: e.httpOptions?.sendTimeout ?? defaultHttpOptions.sendTimeout,
            next_cases: e.httpOptions?.nextCases ?? defaultHttpOptions.nextCases,
            next_tries: e.httpOptions?.nextTries ?? defaultHttpOptions.nextTries,
            next_timeout: e.httpOptions?.nextTimeout ?? defaultHttpOptions.nextTimeout
```